packager/hls/base/simple_hls_notifier.cc

```diff
@@ -41,6 +42,22 @@ bool IsCommonSystemId(const std::vector<uint8_t>& system_id) {
          std::equal(system_id.begin(), system_id.end(), media::kCommonSystemId);
 }
 
+std::string Base64EncodeUri(const std::string& prefix,
+                            const std::string& data) {
+    std::string data_base64;
+    base::Base64Encode(data, &data_base64);
```

Comment:
As mentioned above there is also base 64 encoding for URLs.
I don't quite remember why I didn't do it for the identity key impl, but I think most people prefer base 64 URL encoding. WDYT?